### DISCUSSION TITLE
일반 텍스트에 링크 씌우기??

### DISCUSSION THREAD

**Discussion by disabled-variable** [storycraft/node-kakao#694] [Q&A]
일반 텍스트에 링크를 걸 수 있는 방법이 있나요?

[네이버] [다음] [구글] 이렇게 각각을 한글로 클릭가능한 채팅으로 보내주고, 각 글씨를 클릭했을 때 해당 링크페이지로 이동하도록 하고싶습니다.

**Comment by generalxd** [comment]
불가능합니다.

**Reply by disabled-variable** [reply]
![image](https://user-images.githubusercontent.com/84914862/135212817-75f29d1d-ecf2-4617-817a-8839be7a6e14.png)

인터넷뒤져서 뿌튀업체들 테러하고 다니는거 구해온거라 화질이 별로인점 양해부탁드립니다.
저런식으로 텍스트에 링크를 먹여버리던데 node-kakao 제공기능이 아닐까요?

**Reply by generalxd** [reply]
저건 그냥 텍스트 자체로 링크가 걸리는거 아닌가요 ?
아무 의미없는 링크같습니다.

**Reply by generalxd** [reply]
![image](https://user-images.githubusercontent.com/85729454/135213761-189d924f-4844-4fb4-bc97-b0424e8ebd54.png)

**Reply by disabled-variable** [reply]
클릭하니까 알수없는 링크로 연결이 되더라구요

**Reply by disabled-variable** [reply]
그렇다면 링크값에 제대로된 링크를 넣어주면 이동이 되지않을까요?

**Reply by generalxd** [reply]
네 그냥 카카오톡이 링크로 착각하고 링크달아주는거같네요

**Reply by disabled-variable** [reply]
앞에 # 해시태그때문에 링크로 인식을 했을뿐, 링크값을 넣거나 조작을 해서 생성한건 아닌거같다는 말씀이신가요?

**Reply by generalxd** [reply]
불가능합니다. 저기 보이는 text는 말그대로 String입니다.
서버에서 text로 받고 링크로 렌더링 되는거라 클라이언트 단이 하는역할입니다.

**Reply by generalxd** [reply]
> 앞에 # 해시태그때문에 링크로 인식을 했을뿐, 링크값을 넣거나 조작을 해서 생성한건 아닌거같다는 말씀이신가요?

네. 직접 위메시지 나에게보내기로 처보세요.

**Reply by generalxd** [reply]
> 그렇다면 링크값에 제대로된 링크를 넣어주면 이동이 되지않을까요?

혹 저런 모양의 링크를 눌렀을때 적절한 사이트로 이동이 되는 사례가 있었나요?

**Reply by disabled-variable** [reply]
아직까지는 보지못했습니다. 
어제 우연히 저걸보고 위에 질문내용처럼 응용할 수 있지않을까 싶어서 질문드렸습니다!!

**Reply by ggoek34** [reply]
최근에 텍스트 클릭하면 해당 싸이트로 이동하는  이 방법 이용한 광고 보이네요.

**Reply by benesiaxd** [reply]
혹시 사진 업로드가능한가요 ?
